### PR TITLE
Prepare CircleCI config to run GCP Acceptance tests

### PR DESCRIPTION
**Pull request by usmansaleem**
## PR Description
Prepare CircleCI config to run GCP Acceptance tests

## Documentation

- [x] I thought about documentation and added the `doc-change-required` label to this PR if updates are required.

## Changelog

- [x] I thought about adding a changelog entry, and added one if I deemed necessary.

## Testing

- [x] I thought about testing these changes in a realistic/non-local environment.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -224,6 +224,10 @@ jobs:
           name: Acceptance Tests
           no_output_timeout: 20m
           command: |
+            if [ -n "$GCP_SERVICE_KEY" ]; then
+              echo "$GCP_SERVICE_KEY" | base64 --decode > $HOME/gcp_service_key.json
+              export GOOGLE_APPLICATION_CREDENTIALS=$HOME/gcp_service_key.json
```
Comment (jframe) [inline .circleci/config.yml:229]: Is this just for our ATs with how they work or will users also have to set the `GOOGLE_APPLICATION_CREDENTIALS` environment variable?

Comment (jframe) [reply .circleci/config.yml:229]: Could just have the AT use the GCP_SERVICE_KEY env variable instead?

Comment (usmansaleem) [reply .circleci/config.yml:229]: End users, as well as web3signer developers, have to use "Application Default Credentials ADC) to setup authentication. We are using service user approach which uses service user key and GOOGLE_APPLICATION_CREDENTIALS environment variable. https://cloud.google.com/docs/authentication/provide-credentials-adc

Comment (jframe) [reply .circleci/config.yml:229]: That's not nice. It means we can't use our typical config and env variable approach. And google becomes a special case that is more difficult to use.

Comment (jframe) [reply .circleci/config.yml:229]: For our ATs it's fine though

Comment (siladu) [reply .circleci/config.yml:229]: nit: think it would be clearer if GCP_SERVICE_KEY was called GCP_SERVICE_KEY_BASE64

**Review by jframe — APPROVED** [review]

**Pull request merged**